src/haz3lcore/statics/Grammar.re

```diff
@@ -0,0 +1,415 @@
+open Util;
```

Comment:
I pulled the Grammar out of `TermBase` so that all of the `TermBase` types can stay the same.